spanner/cloud-client/snippets.py

```diff
@@ -598,6 +599,164 @@ def query_data_with_timestamp(instance_id, database_id):
 # [END spanner_query_data_with_timestamp_column]
 
 
+# [START spanner_write_data_for_struct_queries]
+def write_struct_data(instance_id, database_id):
+    """Inserts sample data that can be used to test STRUCT parameters in queries.
```

Comment:
Minor nit: docstrings are inconsistent in terms of spacing/line breaks between functions.

Reply:
Ack, done.